.teamcity/jdks.yaml

```diff
@@ -49,6 +49,18 @@ jdks:
     vendor: "temurin"
     version: "jdk-23.0.1+11"
     sha256: "2400267e4e9c0f6ae880a4d763af6caf18c673714bdee5debf8388b0b5d52886"
+  - params:
```

Comment:
🤔 Why are we not replacing Java 23?

Reply:
We still need Java 23 to run the "latest version" tests for now. I'll clean it up when I can move tests to run on 24.